fe/fe-core/src/main/java/com/starrocks/staros/StarMgrServer.java

```diff
@@ -121,62 +121,6 @@ public void initialize(BDBEnvironment environment, String baseImageDir) throws I
         com.staros.util.Config.STARMGR_IP = FrontendOptions.getLocalHostAddress();
         com.staros.util.Config.STARMGR_RPC_PORT = Config.cloud_native_meta_port;
 
-        // Storage fs type
-        com.staros.util.Config.DEFAULT_FS_TYPE = Config.cloud_native_storage_type;
```

Comment:
com.staros.util.Config.DEFAULT_FS_TYPE = ""

force starmgr not creating those `xxx_from_config` in case the default value is not empty.